### PR TITLE
Added nouns and actions. x10

### PR DESCRIPTION
**Pull request by w1nterl0ng**
Slack username: w1nterl0ng

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun: knife
+//  action 1: Stab a Player
+knife.stab();
+//  action 2: Slash a Player
+knife.slash();
+//  action 3: Hold knife Menacingly
+knife.menace();
```
Comment (matkatmusic) [inline main.cpp]: the player holds the knife menacingly.
pick a different action and revise the pseudo-code
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: Seat
+//  action 1: Move Seat Forward
+seat.moveForward();
+//  action 2: Eject from vehicle
+seat.eject();
```
Comment (matkatmusic) [inline main.cpp]: `seat.ejectFromVehicle()`
- [ ] reise

File: main.cpp [hunk 3]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 2:
-
-//  action 3:
-
-
-
+//  Noun: blender
+//  action 1: Blend Juice
+blender.juice();
```
Comment (matkatmusic) [inline main.cpp]: `juice` is not a verb.
`blender.blendJuice();`
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
+//  action 2: Puree food
+blender.puree();
+//  action 3: spray food around the room
+blender.explode();
```
Comment (matkatmusic) [inline main.cpp]: your pseudo-code doesn't match your plain-english action. 
`blender.sprayFoodAroundTheRoom();`
- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun: Car
+//  action 1: Turn Left
+car.turnLeft();
+//  action 2: Accelerate
+car.accelerate();
+//  action 3: Narrow the distance to the car leading it.
+car.tailgate();
```
Comment (matkatmusic) [inline main.cpp]: your pseudo-code doesn't match your plain-english action.
- [ ] revise

File: main.cpp [hunk 6]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: box
+//  action 1: Move the box
+box.move();
```
Comment (matkatmusic) [inline main.cpp]: Who is moving the box? 
You need to pick actions that the Noun performs.  This action is performed by something other than the Noun ONTO the noun.
- [ ] revise

File: main.cpp [hunk 7]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
+//  action 2: Rotate the Box clockwise
+box.rotateClockwise();
+//  action 3: Flip the box
+box.flipX();
```
Comment (matkatmusic) [inline main.cpp]: You need to pick actions that the Noun performs.  This action is performed by something other than the Noun ONTO the noun.
- [ ] revise

File: main.cpp [hunk 8]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: knife
+//  action 1: Stab a Player
+knife.stab();
+//  action 2: Slash a Player
+knife.slash();
```
Comment (matkatmusic) [inline main.cpp]: `knife.stabPlayer();`
`knife.slashPlayer();`
- [ ] revise

File: main.cpp [hunk 9]
```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun: dice
+//  action 1: Roll the dice
+dice.roll();
+//  action 2: pick up the dice
+dice.pickup();
+//  action 3: Drop the Dice
+dice.drop();
```
Comment (matkatmusic) [inline main.cpp]: You need to pick actions that the Noun performs.  These action is performed by something or someone other than the Noun ONTO the noun.
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
This concept of picking nouns and actions that the noun performs is fundamental to writing code that is easy to read and understand. 
A lot of your submissions have actions that are performed ON the noun, which is not aligning with this concept of choosing actions that the nouns perform. 
Consider: 
```
noun: car
action: drive car
```
the car does not drive the car.   A person drives the car. 
A correct revision would be:
```
noun: car
action: carry passengers
noun: person
action: drives car
```
the car carries passengers. 
the person drives the car.

This concept is extremely important, but it is directly tied to how proficient your English Language Skills are.  If english is not your native language, then these concepts of nouns and actions might not align with the grammar and sentence structure your native language uses.  Keep this in mind when you make your revisions if English is not your native language. 
```
[Noun (the sentence subject)] [an action that the noun performs (verbs)]
```
- [ ] revise

**Comment by w1nterl0ng** [comment]
ok, all changes done, ready for second review

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**